### PR TITLE
[Feat] : Tags CRUD Funtionality 

### PR DESCRIPTION
**Pull request by t007rushi**
Tags : 
Add a Tag
remove a Tag
on Editor as well as NoteCard

It created a separate Label option in the sidebar Dynamically

recording

<details><summary>Browser metadata</summary>

```
Path:      /notes
Browser:   Chrome 99.0.4844.84 on Windows 10
Viewport:  1536 x 775 @1.25x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-10--monkeep.netlify.app%2Fnotes&os=Windows&os_version=10&browser=Chrome&browser_version=99.0&resolution=1536x775&speed=1&start=true&ref=netlify-source)

</details>

[Open Deploy Preview](https://deploy-preview-10--monkeep.netlify.app/notes) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=624be6b5c4f1d10008cd0632&commentID=624bebdc2d41336e588b0278&resolution=resolved)

### CODE REVIEW

File: src/Layout/Main/Main.jsx [hunk 1]
```diff
@@ -48,8 +49,10 @@ export const Main = () => {
             }
           ></Route>
           <Route path="/archives" element={<ArchivesPage />}></Route>
-          <Route path="/labels" element={<></>}></Route>
+
+          <Route path="/labels/:label" element={<LabelsPage />}></Route>
           <Route path="/trash" element={<TrashPage />}></Route>
+          <Route path="/search" element={<></>}></Route>
```
Comment (ultronjs) [inline src/Layout/Main/Main.jsx:55]: What is this route for??

File: src/components/Note/Note.jsx [hunk 2]
```diff
@@ -18,9 +19,11 @@ const breakpointColumnsObj = {
 };
 
 export const Note = ({ arr, heading }) => {
-  const { togglePin, Change_color, inTrash, removeFromnote } = useNotes();
+  const { togglePin, Change_color, inTrash, removeFromnote, tagUpdate } =
```
Comment (ultronjs) [inline src/components/Note/Note.jsx:22]: could be consistent in naming

**Review by ultronjs — APPROVED** [review]
Added some comments otherwise,
Code looks good
Functionality is also Working Fine
Good to merge.

File: src/Layout/Main/Main.jsx (hunk 1 above)
Comment (t007rushi) [reply src/Layout/Main/Main.jsx:55]: Future ref for Search feat

File: src/constants/notesConstant.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
-export const PIN ="PIN"
-export const SET_TITLE ="SET_TITLE"
-export const ADD_TAG ="ADD_TAG"
-export const SET_COLOR ="SET_COLOR"
-export const CLEAR_INPUTS ="CLEAR_INPUTS"
\ No newline at end of file
+export const PIN = "PIN";
```
Comment (munjalSaksham71) [inline src/constants/notesConstant.js:1]: Great that you have create constants

**Review by munjalSaksham71 — APPROVED** [review]
Code is understandable and functionality working as expected! Great work👍🏻

**Pull request merged**